### ISSUE TITLE
Use Firefox Version when serializing Rapid Recipe

### ISSUE THREAD

**Issue by jaredlockhart** [mozilla/experimenter#2955]
Right now, we would pull the JEXL out of the audience choice, but if we also want to filter on Firefox Version we need to add an additional JEXL substatement.

**Comment by jaredlockhart** [comment]
This is blocked on hooking Firefox Version up to the UI in https://github.com/mozilla/experimenter/issues/2965

**Comment by jaredlockhart** [comment]
Closing as a duplicate of https://github.com/mozilla/experimenter/issues/2965
